go.mod

```diff
@@ -26,11 +26,11 @@ require (
 	github.com/onsi/gomega v1.10.1
 	github.com/prometheus/tsdb v0.10.0 // indirect
 	github.com/renproject/aw v0.4.0-9
-	github.com/renproject/darknode v0.5.3-0.20201105235245-6c826ec308fa
+	github.com/renproject/darknode v0.5.3-0.20210104053639-3cf5558a22ae
```

Comment:
This version is a bit out-of-date, can we update to use the latest one on `fix/account-payload`?